packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -1477,6 +1492,49 @@ function commitSuspenseComponent(finishedWork: Fiber) {
   }
 }
 
+function getChildrenOfSuspenseBoundary(suspenseBoundary) {
```

Comment:
For updates, both this function and `hideOrUnhideAllChildren` will be called. Can you refactor so that `hideOrUnhideAllChildren` does both during updates? So we don't have to traverse twice.